public/scss/includes/variables.scss

```diff
@@ -0,0 +1,2 @@
+// Fonts
+$font-metropolis: "Metropolis", Arial, Helvetica, sans-serif;
```

Comment:
Does this need some `$font-inter` with fallbacks? Or is that done elsewhere already?

Reply:
This was an alternate to the the [`.ff-Met` class](https://github.com/mozilla/blurts-server/blob/main/public/css/app.css#L118-L120), which had no fallbacks. 

Looking at the body class `font-family` property: 
```
font-family: "Inter UI", -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Open Sans', 'Helvetica Neue', sans-serif;
```